src/main/java/stringcalculator/domain/Delimiter.java

```diff
@@ -0,0 +1,47 @@
+package stringcalculator.domain;
+
+import static stringcalculator.domain.Validator.validatePositiveNumber;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Delimiter {
+
+    private final static String CUSTOM_DELIMITER = "//(.)\n(.*)";
+    private final static String[] DEFAULT_DELIMITER = new String[]{",", ":"};
+
+    private static final List<String> delimiters = new ArrayList<>(
+        Arrays.asList(DEFAULT_DELIMITER));
+
+    public Delimiter() {
+    }
+
+    public static String findExpression(String input) {
+        Pattern pattern = Pattern.compile(CUSTOM_DELIMITER);
+        Matcher matcher = pattern.matcher(input);
+        if (matcher.find()) {
+            addCustomDelimiter(matcher.group(1));
+            return matcher.group(2);
+        }
+        return input;
+    }
+
+    private static void addCustomDelimiter(String delimiter) {
+        delimiters.add(delimiter);
+    }
+
+    public static List<Integer> extractNumberFromExpression(String input) {
+        return convertOperandType(Arrays.asList(input.split(String.join("|", delimiters))));
```

Comment:
하나의 라인에서 많은 일을 하고 있는 것 같아요!
이런 경우 가독성이 떨어질 수 있을 것 같은데요~!
하나의 라인을 여러개의 라인으로 분리해보면 어떨까요?

```suggestion
        final String joinedDelimiters = String.join("|", delimiters);
        final List<String> splitResults = Arrays.asList(input.split(joinedDelimiters));
        return convertOperandType(splitResults);
```